sphinx/user-guide/installation.rst

```diff
@@ -0,0 +1,427 @@
+     echo "*/10 * * * * svn update -q ${DEST}/*"
+   } >>'crontab.tmp'
+   crontab 'crontab.tmp'
+   rm 'crontab.tmp'
+
+   # Finally add the root level "meta-path" setting to site's "rose.conf"
+   # E.g. if Rose is installed under "/opt/rose/":
+   {
+     echo '[]'
+     echo "meta-path=${DEST}"
+   } >>'/opt/rose/etc/rose.conf'
+
+.. tip::
+   See also :ref:`app-meta-loc`.
+
+
+Configuring Rose Bush
+---------------------
+
+Rose Bush provides an intranet web service at your site for users to
+view their suite logs using a web browser. Depending on settings at
+your site, you may or may not be able to set up this service.
+
+You can start an ad-hoc Rose Bush web server by running::
+
+   setsid /path/to/rose/bin/rose bush start 0</dev/null 1>/dev/null 2>&1 &
+
+You will find the access and error logs under ``~/.metomi/rose-bush*``.
+
+Alternatively you can run the Rose Bush web service under Apache
+``mod_wsgi``. To do this you will need to set up an Apache module
+configuration file (typically in ``/etc/httpd/conf.d/rose-wsgi.conf``)
+containing the following (with the paths set appropriately)::
+
+   WSGIPythonPath /path/to/rose/lib/python
+   WSGIScriptAlias /rose-bush /path/to/rose/lib/python/rose/bush.py
+
+Use the Apache log at e.g. ``/var/log/httpd/`` to debug problems.
+See also Configuring Rosie Server below.
+
+.. TODO ---
+
+
+Configuring a Rosie Server
+--------------------------
+
+You should only need to configure and run your own Rosie service if you do
+not have access to Rosie services on the Internet, or if you need a
+private Rosie service for your site. Depending on settings at your
+site, you may or may not be able to set up this service.
+
+You will need to select a machine to host the Subversion repositories.
+This machine will also host the web server and databases.
+
+Login to your host, create one or more Subversion FSFS repositories.
+
+If you want to use FCM for your version control, you should set a
+special property on the repository to allow branching and merging
+with FCM in the Rosie convention. For example, if your repository
+is served from ``HOST_AND_PATH`` (e.g. ``myhost001/svn-repos``) with
+given repository base name NAME (e.g. ``roses_foo``), change into a
+new directory and enter the following commands::
+
+   svn co -q "svn://${HOST_AND_PATH}/${NAME}/"
+   svn ps fcm:layout -F - "${NAME}" <<'__FCM_LAYOUT__'
+   depth-project = 5
+   depth-branch = 1
+   depth-tag = 1
+   dir-trunk = trunk
+   dir-branch =
+   dir-tag =
+   level-owner-branch =
+   level-owner-tag =
+   template-branch =
+   template-tag =
+   __FCM_LAYOUT__
+   svn ci -m 'fcm:layout: defined.' "${NAME}"
+   rm -fr "${NAME}"
+
+Add the following hook scripts to the repository:
+
+* pre-commit::
+
+     #!/bin/bash
+     exec /path/to/rose/sbin/rosa svn-pre-commit "$@"
+
+* post-commit::
+
+      #!/bin/bash
+      exec /path/to/rose/sbin/rosa svn-post-commit "$@"
+
+You should replace ``/path/to/rose/`` with the location of your Rose
```

Comment:
I think angle bracket notation (e.g. <path>) would be less ambiguous as a placeholder, but naturally it is up to the writer!

Reply:
Agreed!